src/PureBlazor.Components/PureBlazor.Components.csproj

```diff
@@ -23,6 +23,14 @@
     <PackageOutputPath>..\packages</PackageOutputPath>
   </PropertyGroup>
 
+  <ItemGroup>
```

Comment:
just confirming, is this still relevant? might be able to remove it.

Reply:
you are correct, is not needed